src/Raven.Server/Documents/Sharding/ShardedDatabaseContext.cs

```diff
@@ -139,6 +150,242 @@ private void UpdateConfiguration(Dictionary<string, string> settings)
             Configuration = DatabasesLandlord.CreateDatabaseConfiguration(ServerStore, DatabaseName, settings);
         }
 
+        public static void FillShardingConfiguration(ServerStore serverStore, AddDatabaseCommand addDatabase, ClusterTopology clusterTopology)
```

Comment:
does not look like an appropriate place for this?

Reply:
moved to `ServerStore.Sharding`